New index.html design & added docu links

theRealImy opened the pull request:
The new index page looks like:
![Screenshot 2021-11-09 at 16 17 49](https://user-images.githubusercontent.com/4144203/140952384-b21a60dd-2e00-4cdc-98c0-d246a620a8f6.png)

After logged in a message with the WebID is displayed under the header, like:
![grafik](https://user-images.githubusercontent.com/4144203/140952718-2a179fe6-f714-4a7a-b0c1-77ef6e2a9344.png)

Remark: this is from the template so please do not mind the {{..}} fields they will be replaced by pod providers.

ewingson commented:
@theRealImy @jeff-zucker @bourgeoa 

I'm not sure what will happen without the removed javascript-code in the `index.html`
we will have to put hands *manually* to the file, or did I misunderstand anything and the template will be processed in any way ?

in the suggested changes/PR I'm afraid we will lose the functionality of the two buttons. but the impression of the look&feel is great. correct me if I'm too naive.

ewingson commented:
everything back, the code is now in common/js/index-buttons.js as I just see.......

ewingson reviewed (commented):
LGTM

bourgeoa commented:
@theRealImy 
Seems nice. I must test it.

I like the WebId link appearing under the header.

A few more comments.  Your changes removes the no-brand concept :
- this a generic index.html and nothing specific should be there
- anyone could have his own brand logo including in the header
- links should be minimal : your additions give too much focus on solid
- remove the link to create your own server. It is not even on NSS but on CSS

bourgeoa commented on common/js/index-buttons.js:
```diff
@@ -0,0 +1,42 @@
+        window.localStorage.removeItem(keyname);
+        //document.getElementById('prototype').style.display = "none";
+        document.getElementById('loggedIn').style.display = "block";
+        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is: <a href="${webId}">&lt;${webId}&gt</a>. Visit your profile to log into your Pod.</p>`;
```
```suggestion
        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>. Visit your profile to log into your Pod.</p>`;
```

bourgeoa commented:
Trying locally it appears : 
- that some CSS have changed and now the register pane appear in too small characters.
- I also noticed that the displayed WebId has `<webId>`. We must remove the `<>`

theRealImy commented:
I changed the texts and made the logo configurable. The page now looks like:
![grafik](https://user-images.githubusercontent.com/4144203/141104183-2913f272-d93a-4165-a64c-dfbc25242d87.png)
and after login:
![grafik](https://user-images.githubusercontent.com/4144203/141104021-e8cfbf69-4ecb-4e53-9ff7-e85a533296d3.png)
I also fixed the problem of the font size on the register page.

bourgeoa reviewed (approved):
@timea I tried locally and there is problem. Did you miss to push ?

TallTed commented on common/js/index-buttons.js:
```diff
@@ -0,0 +1,41 @@
+        if( webId ) {
+        window.localStorage.removeItem(keyname);
+        document.getElementById('loggedIn').style.display = "block";
+        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>. Visit your profile to log into your Pod.</p>`;
```
Will make the instruction clearer.
```suggestion
        document.getElementById('loggedIn').innerHTML = `<p>Your WebID is : <a href="${webId}">${webId}</a>.</p> <p>Visit your profile to log into your Pod.</p>`;
```

bourgeoa reviewed (approved):
LGTM

ewingson commented:
personally I find a little margin in the dt element better. but that's just perfectionism and a bit opinionated.
